Constrained_triangulation_3/doc/Constrained_triangulation_3/Constrained_triangulation_3.txt

```diff
@@ -3,20 +3,72 @@ namespace CGAL {
 
 \mainpage User Manual
 \anchor Chapter_CT_3
+\anchor userchapterct3
+
 \cgalAutoToc
 \author Laurent Rineau
 
-This chapter describes the ...
+\section Conforming Delaunay Triangulations in 3D
+
+This package implements the construction of a 3D conforming Delaunay triangulation,
+constrained to a set of faces of a surface polygon mesh.
+
+For any Piecewise Linear Complex (PLC) in 3D,
+the algorithm builds a Delaunay triangulation conforming to this PLC.
+The constrained triangulation does not always exist, and it may be necessary
+to add Steiner points to the PLC to make it triangulable.
+
+The problem is described by Cohen-Steiner et al~\cgalCite{cgal:cohen2002conforming},
+and by Si~\cgalCite{cgal:si2008cdt3}.
```

Comment:
Even though WIP
- the `~` is a LaTeX sign for a space, I think it is better to use `&nbsp;` here.